transformJsonToXml/TransformJsonToXml.sh

```diff
@@ -0,0 +1,3 @@
+cd "../"
+cd "service"
+./gradlew run
```

Comment:
Shell scripts are often `cat`ted etc, so having that new line at the end is useful not to mess with people's terminals.
```suggestion
./gradlew run

```